### PR TITLE
Boiler plate code for the storage service

### PR DESCRIPTION
**Pull request by hiendo**

### CODE REVIEW

File: management/src/main/java/com/bitwisekaizen/sdss/config/ApplicationConfig.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        return ds;
+    }
+
+    @DependsOn("flyway")
+    @Bean
+    public LocalContainerEntityManagerFactoryBean entityManagerFactory(DataSource dataSource, JpaVendorAdapter jpaVendorAdapter) {
+        LocalContainerEntityManagerFactoryBean lef = new LocalContainerEntityManagerFactoryBean();
+        lef.setDataSource(dataSource);
+        lef.setJpaVendorAdapter(jpaVendorAdapter);
+        lef.setPackagesToScan("com.bitwisekaizen.storage");
```
Comment (bitwisekaizen) [inline management/src/main/java/com/bitwisekaizen/sdss/config/ApplicationConfig.java]: Should we remain consistent between this and pxeasy - entity?

File: management/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+app.server.port=8080
+app.persistence.jdbcUrl=jdbc:h2:file:~/pxeasy.db
```
Comment (bitwisekaizen) [inline management/src/main/resources/application.properties]: Change name.

File: management/src/test/java/com/bitwisekaizen/sdss/acceptance/AppStartupTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+@SpringApplicationConfiguration(classes = ApplicationConfig.class)
+@WebAppConfiguration
+@IntegrationTest({"server.port:0"})
+// See http://stackoverflow.com/questions/25537436/acceptance-testing-a-spring-boot-web-app-with-testng
+@TestExecutionListeners(inheritListeners = false, listeners = {
+        DependencyInjectionTestExecutionListener.class,
+        DirtiesContextTestExecutionListener.class})
+@Test
+@TestPropertySource(properties = "tail.autoStartup = false", locations = {"classpath:test.properties"})
```
Comment (bitwisekaizen) [inline management/src/test/java/com/bitwisekaizen/sdss/acceptance/AppStartupTest.java]: Don't need this now.

Comment (hiendo) [inline management/src/test/java/com/bitwisekaizen/sdss/acceptance/AppStartupTest.java]: Still need this to make sure my plumbing is correct with some of the changes I made.

File: management/src/main/resources/application.properties (hunk 2 above)
Comment (hiendo) [inline management/src/main/resources/application.properties]: nice catch.

**Comment by bitwisekaizen** [comment]
Looks good.  A couple of comments and then merge.

**Pull request merged**